frontend/src/pages/notebookController/screens/admin/NotebookActions.tsx

```diff
@@ -34,10 +41,10 @@ const NotebookActions: React.FC<ServerStatusProps> = ({ data }) => {
   return (
     <>
       <ActionsColumn items={rowActions} />
-      {showModal && (
+      {showModal && notebookLinkLoaded && (
         <StopServerModal
           notebooksToStop={notebooksToStop}
-          link="#"
+          link={notebookLinkError || !notebookLink ? '#' : notebookLink}
```

Comment:
```suggestion
          link={notebookLink}
```
I dont think we need to show the error in the link when populating it.

Also, I think we can design this component better by changing the type to `link?: string;` in stop server modal to accept undefined. So we remove the check for hardcoded `'#'` and check for undefined instead (fix the checking function), and in `NotebookServer.tsx`, we can do

```
<StopServerModal
                  notebooksToStop={notebooksToStop}
                  isDeleting={isDeleting}
                  link={link !== '#' ? link : undefined}
                  onNotebooksStop={onNotebooksStop}
                />
```

Reply:
Ah i see, i was probably too tired to read the ternary correctly lol